src/App.tsx

```diff
@@ -1,18 +1,203 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useContext,
+  useEffect,
+  useState,
+  useRef,
+} from 'react';
+
+import classNames from 'classnames';
 import { AuthContext } from './components/Auth/AuthContext';
+import { Loader } from './components/Loader';
+import { TodoList } from './components/TodoList';
+import { Notification } from './components/Notification';
+
+import {
+  createTodo,
+  getTodos,
+  removeTodo,
+  changeTodo,
+} from './api/todos';
+
+import { Todo } from './types/Todo';
+import { Filter } from './types/Filter';
 
 export const App: React.FC = () => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const user = useContext(AuthContext);
   const newTodoField = useRef<HTMLInputElement>(null);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [filterKind, setFilterKind] = useState<Filter>(Filter.All);
+  const [notification, setNotification] = useState<string>('');
+
+  const [title, setTitle] = useState('');
+  const [isAdding, setIsAdding] = useState(false);
+  const [temporaryTodoTitle, setTemporaryTodoTitle] = useState('');
+  const [loaderForTodo, setLoaderForTodo] = useState<number[]>([]);
+  const completedTodos = todos.filter(todo => todo.completed);
+
+  useEffect(() => {
+    if (user) {
+      getTodos(user.id).then(todosFromServer => {
+        setTodos(todosFromServer);
+        setIsLoading(false);
+      })
+        .catch(() => {
+          setIsLoading(false);
+          setNotification('Cannot load todos from server');
+        });
+    }
```

Comment:
It's better to move async function to a separate function and just call it in useEffect